src/core/u-compress.c

```diff
@@ -355,9 +406,10 @@ REBSER *Decompress(
 
     // Since the initialization succeeded, go ahead and make the output buffer
     //
-    REBSER *output = Make_Binary(buf_size);
+    REBSER *output = Make_Binary(sizeof(REBSER*) + buf_size);
+    *cast(REBSER**, BIN_HEAD(output)) = output; // self-referential prefix
     strm.avail_out = buf_size;
-    strm.next_out = BIN_HEAD(output);
+    strm.next_out = BIN_HEAD(output) + sizeof(REBSER*);
 
```

Comment:
It may seem a good idea to set the bias at the beginning and then all the code can be made simpler...but there's some problem in that series expansion and reallocation will mess with it.  If you have some bias in a series and expand it for Rebol_Realloc(), and there's a new allocation, that allocation won't have any bias.

So the code would have to get its hands dirty a bit lower-level than I felt like risking; but perhaps there could be accommodation of it in the future ("preserved bias?")